packages/KIcon/icons/icn-danger-circle-outline.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
Did you run this through https://jakearchibald.github.io/svgomg/ ?

Reply:
@chrisvdp I didn't know this was a thing. How would you recommend using it? There are a lot of toggle options.

Reply:
I think default is fine https://kongponents.konghq.com/contributing/adding-icons-to-kicon.html